core/src/server/rpc_module.rs

```diff
@@ -843,7 +843,7 @@ impl SubscriptionSink {
 		}
 
 		let msg = self.build_message(result)?;
-		Ok(self.inner_send(msg))
+		Ok(self.inner.send_raw(msg).is_ok())
```

Comment:
Just to check; we don't care about checking that `self.is_active_subscription()` is true any more?

Reply:
Aah I see; is_closed has that check now!